src/components/Menu/Menu.js

```diff
@@ -13,10 +13,11 @@ class Menu extends Component {
    */
   render() {
     if (this.props.menuFetch.fulfilled) {
+      const source = this.props.source;
       return (
         <nav className="menu--footer">
           <ul className="menu" id={`${this.props.type}-menu`}>
-            {this.props.menuFetch.value.map((item, key) => <MenuLink item={item} key={key} />)}
+            {this.props.menuFetch.value.map((item, key) => <MenuLink baseUrl={source} item={item} key={key} />)}
           </ul>
```

Comment:
So this will show something like `http://pr-292-ip25kiy-3g6y4v7pqt6nk.eu.platform.sh/contact-us` on the website (i.e: Frost) ?

Reply:
Yep, merge it and give it a go.

Reply:
I need production url first ;)

Reply:
It's coming

Reply:
Production urls will be `https://www.sportrelief.com/` , `https://www.comicrelief.com/`, etc

Reply:
to return json?

Reply:
Just to clarify, if the api endpoint of production are  `https://www.sportrelief.com/` , `https://www.comicrelief.com/` that's totally fine with me.

Reply:
Yep, thats how it will work